client/src/components/RelatedItems/relatedItemsData.js

```diff
@@ -1,3 +1,131 @@
-module.exports = {
+const relatedItemsdata = {
+  currentProductInfo: {
```

Comment:
It actually needs currentProductInfo for the ComparisonModal component. 
It may seem unnecessary now because I decided to skip that component implementation.